src/Nox.Generator/Presentation/Api/OData/EntityControllerGenerator.cs

```diff
@@ -240,7 +243,8 @@ private static void GeneratePut(Entity entity, CodeBuilder code, NoxSolution sol
     private static void GeneratePatch(Entity entity, string entityName, string pluralName, CodeBuilder code, NoxSolution solution)
     {
         // Method Patch
-        code.AppendLine($"public async Task<ActionResult> Patch({PrimaryKeysFromRoute(entity, solution)}, [FromBody] Delta<{entityName}UpdateDto> {entity.Name.ToLowerFirstChar()})");
+        code.AppendLine($"[EnableQuery]");
```

Comment:
same here, this Enable Query brings some security issues and complexity that for now we can avoid on this endpoints